go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/Shopify/sarama
 
+go 1.13
```

Comment:
@varun06 I'm not 100% this is needed (this line added by `go mod`), do you know if it's something that will break 1.11/1.12 compatibility?

According to https://golang.org/doc/go1.13 it shouldn't but 🤔:

> If your code uses modules and your go.mod files specifies a language version, be sure it is set to at least 1.13 to get access to these language changes. You can do this by editing the go.mod file directly, or you can run go mod edit -go=1.13.

Reply:
yeah, I treat it like a ceiling for go features. So it shouldn't impact anything with backward compatibility.